docs/howtos/audit-scanner.md

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_label: "Audit Scanner Installation"
+title: "Audit Scanner Installation"
+---
+
+# Audit Scanner installation
+
+Starting from version `v1.7.0`, Kubewarden introduces a new component called
+the Audit Scanner. This scanner evaluates whether the resources running in the
+cluster comply with the deployed policies. It is particularly useful when
+policies are deployed after the resources. The Audit Scanner allows operators
+to detect resources deployed before the policies that violate the validations
+performed by the policies.
+
+## Installation
+
+To install and use the Kubewarden Audit Scanner, please follow these steps:
+
+1. Install the `kubewarden-controller` Helm chart, ensuring you have version
+   `v1.7.0` or higher. Remember to enable the audit scanner component.
+
+   ```console
+   helm install kubewarden-crds kubewarden/kubewarden-crds
```

Comment:
The step says install `kubewarden-controller`, but here we are installing the crds. I suppose we want a step 1 that installs the crds, were we say that they by default come with the PolicyReports CRDs, and we have the `::caution` paragraph. And then a step 2 for installing the kubewarden controller, enabling the audit scanner, and talking about its defaults.